### DISCUSSION TITLE
Select GBC games will not load

### DISCUSSION THREAD

**Discussion by dapperdyln** [eduardofilo/RG350_adam_image#252] [Q&A]
My RG280v broke and while waiting on my screen replacement to arrive I decided to order a RG300X as a temporary replacement. I re-routed the kernel to the RG300X and inserted into the device. All is well, games are working and save files are maintained. When I tried to boot up Pkmn Gold, which I have previously played and have save files for, will not boot. The boot logo will load and then the screen will go to a blank white. Tried doing away with the BIOs and that did nothing. All other games will load except this one. When I load on the Gearboy emulator, it boots no problem but this emulator to me isn't as appealing as the Gambatte emulator. Any ideas on what the problem might be and how to fix it?

**Comment by eduardofilo** [comment]
Hello, I have been able to reproduce the problem you mention. It happens even in reverse. I have switched from a RG3350M (I don't have the RG300X but it has the same screen resolution, so I've tried with it) to a RG280V and the blank white screen appeared. I managed to solve it by deleting a couple of files that are generated in the `~/.retroarch/saves` directory.

```
rg280v:~/.retroarch/saves $ ls
Pokemon - Edicion Gold.rtc  Pokemon - Edicion Gold.srm
rg280v:~/.retroarch/saves $ rm *
```

The console in that state with the blank white screen is almost hanged. You can neither close RetroArch with Select+Start nor force it to close with Power+Select. Fortunately it does respond via SSH and I was able to launch the reboot command.

Try deleting those files with Commander for example.

**Reply by eduardofilo** [reply]
Don't be afraid of losing game progress. It is stored in the `~/.retroarch/states` directory. What is in `saves` must be a kind of cache that helps RetroArch to resume the cores sessions.

**Reply by dapperdyln** [reply]
Pardon my ignorance, but when you say commander what are you referring to?

**Reply by dapperdyln** [reply]
I am generally lost. I'm not sure where to go to solve this issue. Do I solve this on the RG300X itself or on my PC? Forgive me, I'm still fairly new to this world of retro handhelds.

**Reply by neilswann80** [reply]
`Commander` is a File Explorer app on the device, it'll be in the `Apps` section of `Apps and Games`

Remove everything in: `/media/data/local/home/.retroarch/saves`
(as @eduardofilo stated, these aren't game save-states they're retroarch cached settings, so you won't lose game progress)

Second option...
If you can't access the system on your device at all, you should be able to connect the device to your PC and remove them via [SSH](https://github.com/eduardofilo/RG350_adam_image/wiki/En:-4.-System-access)
